DataManagementSystem/Client/FTSJob.py

```diff
@@ -729,9 +729,15 @@ def toSQL( self ):
 
     :return: str with SQL fragment
     """
-    colVals = [ ( "`%s`" % column, "'%s'" % value if type( value ) in ( str, datetime.datetime ) else str( value ) )
-                for column, value in self.__data__.items()
-                if value and column not in  ( "FTSJobID", "LastUpdate" ) ]
+    colVals = []
```

Comment:
The type of the string must be evaluated as isinstance( value, basestring ). We should all take this on board. In the replaced code the type of value was compared to "str" which is not taking into account Unicode strings. Again, this is what happened in my tests.